tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -40,6 +40,13 @@
     private static $_queryCacheImpl = null;
 
     /**
+     * The result cache shared between all functional tests.
+     *
+     * @var \Doctrine\Common\Cache\Cache|null
+     */
+    private static $_resultCacheImpl = null;
```

Comment:
If we introduce new properties, we shouldn't use the prefix... Also, the fact that this is a static smells a lot: what if it is accidentally not unset between test cycles? Is it supposed to always be set?